JitsiConference.js

```diff
@@ -1371,28 +1373,28 @@ JitsiConference.prototype._setNewVideoType = function(track) {
     const videoTypeTagName = 'videoType';
 
     // if video type is camera and there is no videoType in presence, we skip adding it, as this is the default one
-    if (track.videoType !== VideoType.CAMERA || this.room.getFromPresence(videoTypeTagName)) {
+    if (trackVideoType !== VideoType.CAMERA || this.room.getFromPresence(videoTypeTagName)) {
         // we will not use this.sendCommand here to avoid sending the presence immediately, as later we may also set
         // and the mute status
-        return this.room.addOrReplaceInPresence(videoTypeTagName, { value: track.videoType });
+        return this.room.addOrReplaceInPresence(videoTypeTagName, { value: trackVideoType });
     }
 
     return false;
 };
 
 /**
  * Sets mute status.
- * @param localTrack
+ * @param mediaType
  * @param isMuted
  * @param <tt>true</tt> when presence was changed, <tt>false</tt> otherwise.
  * @private
  */
-JitsiConference.prototype._setTrackMuteStatus = function(localTrack, isMuted) {
+JitsiConference.prototype._setTrackMuteStatus = function(mediaType, isMuted) {
```

Comment:
can you keep the local track argument here? it will be needed for multi stream support

Reply:
I added it back, but it is unused for now.

There is a case where you set it back to default, there is no track.